### PR TITLE
Bug 1506791 - drag reorder and delete is crashing. let's not.

### PR DESCRIPTION
**Pull request by garvankeeley**
The fix is to ignore the drag and reload the collection view.
It would be nice-to-have if the drag could be animated back to exit the
drag state, but Apple APIs do not expose that level of control. All we
can do is ignore the drag.

I was trying to do something similar before this patch (by just ignoring the drag), but that was insufficient, the collection view needs to be reloaded fully.

### CODE REVIEW

**Comment by farhanpatel** [comment]
Looks good. But tests arent compiling

**Comment by garvankeeley** [comment]
Oops, is 100% more buildy now.

File: Client/Frontend/Browser/TabDisplayManager.swift [hunk 1]
```diff
@@ -390,11 +392,17 @@ extension TabDisplayManager: TabManagerDelegate {
         }
     }
 
-    func tabManager(_ tabManager: TabManager, willRemoveTab tab: Tab) {
-        isDragging = false
-    }
+    func tabManager(_ tabManager: TabManager, willAddTab tab: Tab) {}
```
Comment (farhanpatel) [inline Client/Frontend/Browser/TabDisplayManager.swift]: willAddTab and willRemoveTab were only being used by TabDisplayManager. We can remove them completely. Also make sure to remove any tests that use them

Comment (garvankeeley) [reply Client/Frontend/Browser/TabDisplayManager.swift]: Less code is good! will remove...

**Pull request merged**